src/main/java/org/veupathdb/service/eda/subset/model/db/PlatformUtils.java

```diff
@@ -0,0 +1,36 @@
+package org.veupathdb.service.eda.subset.model.db;
+
+import javax.sql.DataSource;
+import java.sql.SQLException;
+import java.util.Arrays;
+
+public class PlatformUtils {
+
+  public enum DBPlatform {
+    PostgresDB("PostgreSQL"),
```

Comment:
Looks like you can get the same effect (enum for platform) with this code:
```
SupportedPlatform.toPlatform(dataSource.getConnection().getMetaData().getDatabaseProductName())
```
See https://github.com/VEuPathDB/FgpUtil/blob/master/Db/src/main/java/org/gusdb/fgputil/db/platform/SupportedPlatform.java